### PR TITLE
Update GettingStartedGuide.md

### PR DESCRIPTION
**Pull request by iriza7**
* Update GettingStartedGuide.md.

* Fix incorrect links and text.

Relates-To: OLPEDGE-1118.

Signed-off-by: Bautista, Iryna <ext-iryna.bautista@here.com>

### CODE REVIEW

File: docs/GettingStartedGuide.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ The following two authentication approaches are available within your applicatio
 
 2. Using your **app access key id** and **secret access key**, which you can find on platform's [Apps & Keys](https://platform.here.com/profile/apps-and-keys) page. For more details on how to create your authentication and authorization client for retrieving HERE tokens, see [Authentication and Authorization Developer's Guide](https://developer.here.com/olp/documentation/access_control/topics/introduction.html).
 
-After you set up the credentials, you can then use them to access HERE OLP SDK libraries.
+After you set up the credentials, you can use them to access HERE OLP SDK libraries.
```
Comment (HalynaDumych) [inline docs/GettingStartedGuide.md]: Change to:
"After you set up the credentials, you can use them to access the HERE OLP SDK libraries."

File: docs/GettingStartedGuide.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ The following two authentication approaches are available within your applicatio
...
 
 ## <a name="components"></a> Available Components
```
Comment (HalynaDumych) [inline docs/GettingStartedGuide.md]: Comment to lines 40-45
Please change to:
"The OLP SDK for TypeScript package contains four independent modules that focus on a specific set of responsibilities:

- `olp-sdk-authentification`: gets OAuth2 bearer tokens used to confirm OLP requests.
- `olp-sdk-dataservice-api`: retrieves data from OLP.
- `olp-sdk-dataservice-read`: retrieves layer and partition data from OLP catalogs.
- `olp-sdk-fetch`: adds a subset of the [fetch](https://fetch.spec.whatwg.org/) API for [Node.js](https://nodejs.org/).

File: docs/GettingStartedGuide.md [hunk 3]
```diff
@@ -50,7 +50,7 @@ HERE OLP SDK consists of specific libraries, each with distinct functionality.
```
Comment (HalynaDumych) [inline docs/GettingStartedGuide.md]: Comment to line 49
Articles are missing.
Change to:
"[Versioned Layer Read example](./examples/nodejs-read-versioned-layer.md) shows how to retrieve layer metadata, partition metadata, and partition data from a versioned layer using the OLP SDK for TypeScript."

File: docs/GettingStartedGuide.md [hunk 4]
```diff
@@ -50,7 +50,7 @@ HERE OLP SDK consists of specific libraries, each with distinct functionality.
 
 [Volatile Layer Read example](./examples/nodejs-read-volatile-layer.md) shows how to retrieve layer metadata, partition metadata, and partition data from volatile layer using OLP SDK for TypeScript.
```
Comment (HalynaDumych) [inline docs/GettingStartedGuide.md]: Articles are missing.
Change to:
"[Volatile Layer Read example](./examples/nodejs-read-volatile-layer.md) shows how to retrieve layer metadata, partition metadata, and partition data from a volatile layer using the OLP SDK for TypeScript."

File: docs/GettingStartedGuide.md [hunk 5]
```diff
@@ -50,7 +50,7 @@ HERE OLP SDK consists of specific libraries, each with distinct functionality.
...
 
-## <a name="CatalogClient"></a> CatalogClient
+## <a name="catalog-client"></a> CatalogClient
 
 The main entry point to the OLP is `CatalogClient`. This class provides a high-level interface for retrieval of `OLP` catalog data and configuration, and defines the following operations:
```
Comment (HalynaDumych) [inline docs/GettingStartedGuide.md]:
```suggestion
The main entry point to OLP is `CatalogClient`. This class provides a high-level interface to retrieve `OLP` catalog data and configuration, and defines the following operations:
```

File: docs/GettingStartedGuide.md [hunk 6]
```diff
@@ -50,7 +50,7 @@ HERE OLP SDK consists of specific libraries, each with distinct functionality.
...
```
Comment (HalynaDumych) [inline docs/GettingStartedGuide.md]: Comment to lines 57-58: add full stops at the end of the list items.

Comment (HalynaDumych) [reply docs/GettingStartedGuide.md]: Comment to line 60
Change to:
"For more details on how to use the `CatalogClient` object, see the [catalog usage example](./examples/nodejs-examples.md)."

**Comment by iriza7** [comment]
@HalynaDumych Fixed review comments in the following MR:
https://github.com/heremaps/here-olp-sdk-typescript/pull/62

**Pull request merged**